src/toil_scripts/gatk_germline/hard_filter.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python2.7
```

Comment:
I think it's easier to read if the independent components of the pipeline are in separate modules.